components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -91,6 +91,7 @@ open class GleanInternalAPI internal constructor () {
      * as shared preferences
      * @param configuration A Glean [Configuration] object with global settings.
      */
+    @JvmOverloads
```

Comment:
The JVM Overlords. 🙈 (sorry, had to make that joke, nothing to see here, move on!)

Reply:
Oh my. But that's a good one :D